### PR TITLE
Vs eng isolate interaction UI code lv 559

### PR DESCRIPTION
**Pull request by akalscheur-bradley**
Moves UI functionality to its own script and subsequently cleaned up the prior functionality for interaction system

### CODE REVIEW

File: Assets/GameAssets/UI/UIScripts/InteractableUI.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class InteractableUI : MonoBehaviour
+{
+    private GameObject _interactUI;
```
Comment (AlecatEnjoyer) [inline Assets/GameAssets/UI/UIScripts/InteractableUI.cs:7]: personally I would serialize this to get the exact interaction UI without having to worry if someone switches the name of the gameobject. I only really use GameObject.FindWithTag("blahBlah")

**Review by AlecatEnjoyer — APPROVED** [review]
I approved just left comment

**Review by astapay — APPROVED** [review]
Get the other comments done and we're good

File: Assets/GameAssets/UI/UIScripts/InteractableUI.cs [hunk 2]
```diff
@@ -0,0 +1,28 @@
+using System.Collections;
```
Comment (ASLH68) [inline Assets/GameAssets/UI/UIScripts/InteractableUI.cs:1]: File Header

File: Assets/GameAssets/UI/UIScripts/InteractableUI.cs [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+using System.Collections.Generic;
+using UnityEngine;
+
+public class InteractableUI : MonoBehaviour
```
Comment (ASLH68) [inline Assets/GameAssets/UI/UIScripts/InteractableUI.cs:5]: Class XML Comment

**Pull request merged**